source/_redirects

```diff
@@ -2352,3 +2352,5 @@
 /integrations/mychevy /more-info/removed-integration 301
 /components/binary_sensor.mychevy /more-info/removed-integration 301
 /components/sensor.mychevy /more-info/removed-integration 301
+/integrations/loopenergy /more-info/removed-integration 301
```

Comment:
We should update the existing redirects for the integration to redirect directly to removed integration.